src/config/options.schema.json

```diff
@@ -1144,6 +1144,39 @@
           },
           "additionalProperties": false
         },
+        "c2po": {
+          "title": "ana.c2po",
+          "type": "object",
+          "properties": {
+            "askbase": {
+              "title": "ana.c2po.askbase",
+              "description": "If true, the C-2PO Analysis uses the MayPointTo query to infer additional disequalities.",
+              "type": "boolean",
+              "default": true
+            },
+            "join_algorithm": {
+              "title": "ana.c2po.join_algorithm",
+              "description": "Which join algorithm 'c2po' should use. Values: 'partition' (default): the more efficient version, it uses only the partition to compute the join. Circular equalities are lost during the join; 'partition': a more precise version, uses the automaton to compute the join.",
```

Comment:
Just quickly skimmed over it since I listened to your presentation and it sounded interesting.
The only thing I saw was that in this description you describe the value `'partition'` twice. That is probably a typo and the second one should be `'precise'`.
Also, I am happy that the taintPartialContext analysis from my bachelor's thesis gets another use here